src/main/java/atmosfears/AtmosFearsBE/database/AirParticulatesRepository.java

```diff
@@ -1,7 +1,31 @@
 package atmosfears.AtmosFearsBE.database;
 
 import atmosfears.AtmosFearsBE.model.AirParticulates;
+import atmosfears.AtmosFearsBE.model.AggregatedParticulates;
+import org.springframework.data.mongodb.repository.Aggregation;
 import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+import java.util.Date;
+import java.util.List;
 
 public interface AirParticulatesRepository extends MongoRepository<AirParticulates, String> {
+    @Query("{'date': {'$gte': ?0, '$lte':?1 }}")
+    List<AirParticulates> findByDateBetween(Date from, Date to);
+
+    @Aggregation(
+            pipeline = {
```

Comment:
This pipeline looks very complex but I will trust you that it works properly as I don't have time to analyze it in-depth.